core/client/fs/src/main/java/alluxio/client/file/cache/store/RocksPageStore.java

```diff
@@ -106,22 +138,80 @@ public void delete(PageId pageId) throws PageNotFoundException {
   @Override
   public void close() {
     mDb.close();
-    try {
-      FileUtils.deleteDirectory(new File(mRoot));
-    } catch (IOException e) {
-      LOG.warn("Failed to clean up rocksDB root directory.");
-    }
   }
 
   private byte[] getPageKey(PageId pageId) {
```

Comment:
nits:
- since this helper method requires no instance state, possibly turned into a static method. so as the next method. 
- naming: `getKeyFromPageId` and `getPageIdFromKey` to be more explicit on the function? Leaving this up to you.

Reply:
Renamed.